projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -51,6 +53,13 @@ describe('LgCardComponent', () => {
     expect(el.getAttribute('class')).toContain('lg-card');
   });
 
+  it('should have the promotion class if variant is promotion', () => {
```

Comment:
The only other variant test is for testing the 'navigation' and I think that is in the correct Describe by its own, because the 'navigation' variant gets set when there is a 'lg-card-navigation-title', and that is checked in the describe `describe('when there is the lg-card-navigation-title'` and I think that is correct. :)

Reply:
Ah okay, thanks for explaining, makes sense :)